### ISSUE TITLE
Search - Task created in a new group chat is shown in search only after a message is sent

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#51412]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9. 0.53-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N/A
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Go to https://staging.new.expensify.com/home
2. Tap fab -- start chat
3. Create a group chat
4. Create a task
5. Navigate to LHN
6. Tap bottom search icon
7. From drop down, select chat
8. Note task created message not displayed
9. Refresh the page
10. Note now also task created message not displayed
11. Go to group chat
12. Send a message
13. Now navigate to chat page as mentioned in step 7
14. Note both message sent and task is shown.

## Expected Result:
Task /message created in a new group chat must be shown in search instantly

## Actual Result:
Task created in a new group chat is shown in search only after a message is sent

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [x] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/b6110f69-51bb-4a50-9b98-538c940c17e3

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by alexpensify** [comment]
@luacmartins and @JmillsExpensify - do you agree that this is a BE issue? After, I'll add the `Internal` label pending yes? Thanks!

**Comment by luacmartins** [comment]
This is expected. We only display `comment` messages in Search. We descoped `tasks` from v2.5 so for now we won't support this.
